.github/workflows/test_tap.yml

```diff
@@ -32,4 +32,4 @@ jobs:
         poetry run mypy . --ignore-missing-imports
     - name: Test with pytest
       run: |
-        poetry run pytest
+        poetry run pytest --capture=no
```

Comment:
I think capture can be useful to debug on the fly in the CI. Any strong reason for silencing?

Reply:
@ericboucher - This one is confusingly named. "Not capturing" means "not suppressing". Added this setting allowed me to observe the log output needed to confirm the auth.

Reply:
🤦